src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -149,8 +149,6 @@ public function delete(Key $key)
         foreach ($this->stores as $store) {
             try {
                 $store->delete($key);
-            } catch (\Exception $e) {
-                $this->logger->notice('One store failed to delete the "{resource}" lock.', array('resource' => $key, 'store' => $store, 'exception' => $e));
             } catch (\Throwable $e) {
```

Comment:
or should we remove this one?
catching any throwable is a code smell to me, that makes debugging harder than needed.
ping @jderusse

Reply:
if confirmed (it should IMHO), to be moved to 3.4.

Reply:
The purpose of this "catch all" was to clean all stores even when one of theme fail with an \Error. 

The exception is silent because: a single node failure shouldn't be considered as a total failure.
This is exactly the purpose of the CombinedStore: provide a reliable virtual store allowing failure of individual store

Maybe the log level should be higher

I'm :+1:  with this PR, as the split `\Exception` and `\Throwable` was to be compatible with php 5.x which is no more required in symfony 4.x (this PR shouldn't be moved to symfony 3.4 which supports 5.5.9)

Reply:
I'd like someone else to chime in here: to me, catching any "Error" should be rare:
- rethrowing
- debug
- cache-related failure

Those are the only three situations where this looks legit to me.
This is the only case in the code base where we shallow them.

Reply:
This is fixed to not catch Error anymore

Reply:
Perfect timing: https://twitter.com/nikita_ppv/status/1042676176836341760